crowbar_framework/app/helpers/nodes_helper.rb

```diff
@@ -20,48 +20,28 @@ def nodes_by_role(role)
     NodeObject.find("roles:#{role}").sort_by(&:alias)
   end
 
-  def piechart_for(group)
-    values = piechart_values(group)
-    tooltip = piechart_tooltip(values)
-
-    content_tag(
-      :span,
-      "",
-      :title => tooltip,
-      "data-piechart" => values.join(","),
-      "data-tooltip" => "true"
+  def link_to_node(node, handle)
+    link_to(
+      node_icon(node) + " " + truncate(node[:alias], length: 25),
+      node_path(handle),
+      title: node[:description]
     )
   end
 
-  def piechart_tooltip(values)
-    [].tap do |result|
-      result.push content_tag(
-        :strong,
-        t("total", count: values.sum, scope: "nodes.index.status_pie")
-      )
-
-      result.push t("ready", count: values[0], scope: "nodes.index.status_pie") if values[0] > 0
-      result.push t("unknown", count: values[1], scope: "nodes.index.status_pie") if values[1] > 0
-      result.push t("unready", count: values[2], scope: "nodes.index.status_pie") if values[2] > 0
-      result.push t("pending", count: values[3], scope: "nodes.index.status_pie") if values[3] > 0
-      result.push t("crowbar_upgrade",
-                    count: values[4],
-                    scope: "nodes.index.status_pie") if values[4] > 0
-    end.join(tag(:br))
-  end
-
-  def piechart_values(group)
-    [].tap do |result|
-      result.push group[:status]["ready"]
-      result.push group[:status]["failed"]
-      result.push group[:status]["unknown"]
-
-      if group[:status]["building"]
-        result.push group[:status]["unready"] + group[:status]["pending"] + group[:status]["building"]
-      else
-        result.push group[:status]["unready"] + group[:status]["pending"]
-      end
-      result.push group[:status]["crowbar_upgrade"]
+  def node_icon(node)
+    return icon_tag "rocket" if node[:alias] == "crowbar"
```

Comment:
That's a pretty bad indicator, you can give any alias to the admin node. You are forced to check `admin?` to see if that is an admin node.